website/docs/for-capacitor/live-updates.md

```diff
@@ -25,3 +25,29 @@ First let's add Live Updates to the main shell application. Modify the `capacito
 - **autoUpdateMethod**: Options would be `none` or `background`. You would choose `none` if you want to call the update method on your own, but the most common choice here would be `background` where the updates happen automatically.
 
 After you have made these changes and done a build the application will begin pulling updates from Appflow using the IDs that you have provided. See the [reference](reference) documentation for methods on performing live updates manually from your application code.
+
+## Self-hosted Live Updates
+
+Portals for Capacitor supports [Self-hosted Live Updates](https://ionic.io/docs/appflow/deploy/setup/self-hosted)! For our customers with strict security requirements, Self-hosted Live Updates includes enhanced security features built on top of Appflow's already secure delivery mechanisms. Follow the instructions in the [Appflow Docs](https://ionic.io/docs/appflow/deploy/setup/self-hosted#code-signing-generate-live-update-signing-keys) to get started. However,  there is no need to install any additional plugins in your app.
```

Comment:
One nitpick here, you have two spaces between 'However,' and 'there is no need...'